packages/server/src/index.ts

```diff
@@ -998,7 +1051,33 @@ export class App {
                       analytic: chatflow.analytic
                   })
 
+            result = typeof result === 'string' ? { text: result } : result
+
+            await this.addChatMessage({
+                role: 'userMessage',
+                content: incomingInput.question,
+                chatflowid: chatflowid,
+                chatType: isInternal ? chatType.INTERNAL : chatType.EXTERNAL,
+                chatId: chatId,
+                memoryType: memoryLabel ? memoryLabel : undefined,
+                sessionId: sessionId ? sessionId : undefined,
+                createdDate: userMessageDateTime
```

Comment:
do we need this? I think its auto created by typeorm using the `CreateDateColumn`